Warsaw_Tram/TramViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  TramViewModel.swift
+//  Warsaw_Tram
+//
+//  Created by Małgorzata Dziubich on 28/06/16.
+//  Copyright © 2016 Małgorzata Dziubich. All rights reserved.
+//
+
+import UIKit
+
+protocol TramViewModelProtocol {
+    
+    func getTramsData(success: [Tram] -> Void, failure: String -> Void) -> Void
```

Comment:
You don't need to explicitly set return Void value if function doesn't return value